srcpkgs/kleopatra/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'kleopatra'
+pkgname=kleopatra
+version=20.12.3
+revision=1
+archs="x86_64"
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
```

Comment:
Remove these template comments

Reply:
Sorry, this is my first build on void linux. I just used to do some aur package for archlinux before. Thank you for your help! :-)